doc/repos-flag-testing.txt

```diff
@@ -0,0 +1,38 @@
+# loading the eups env essentially to get access to the anaconda python
```

Comment:
Would you like it removed?  There are no unit/acceptance tests in the repo and those are notes on what I did to setup a sandbox VM.